README.md

```diff
@@ -142,11 +142,13 @@ The following is a brief explanation of each available command.
   CONTAINER_TIMEOUT_SECONDS=1800 codeclimate analyze // 30 minutes
   ```
 
+  Note: CONTAINER_TIMEOUT_SECONDS is not currently honored in the CLI.
```

Comment:
Ah! I see the issue - your wrapper is out of date. Did you install via homebrew? If so can you run  `brew update && brew upgrade codeclimate`